source/assets/js/playground.ts

```diff
@@ -229,11 +229,20 @@ function setupPlayground() {
 
   function updateURL() {
     const hash = stateToBase64(playgroundState);
-    history.replaceState('playground', '', `#${hash}`);
+    history.replaceState('playground', '', `?#${hash}`);
```

Comment:
@jamesnw Might be worth adding a comment here for why the `?` is used?

Reply:
After investigating, this isn't actually making a difference, so I reverted back to no query.